modules/statics/src/base.ts

```diff
@@ -456,6 +456,14 @@ export const enum UnderlyingAsset {
   'txlm:TST-GBQTIOS3XGHB7LVYGBKQVJGCZ3R4JL5E4CBSWJ5ALIJUHBKS6263644L' = 'txlm:TST-GBQTIOS3XGHB7LVYGBKQVJGCZ3R4JL5E4CBSWJ5ALIJUHBKS6263644L',
 }
 
+/**
+ * This is the curve BitGo signs against with the user, backup and BitGo key.
+ */
+export const enum SupportedKeyCurve {
+  Secp256k1,
```

Comment:
Please assign to the corresponding string, otherwise debugging const enums after compilation becomes difficult.

Reply:
oh, sure